app/helpers/application_ui_helper.rb

```diff
@@ -0,0 +1,31 @@
+module ApplicationUiHelper
+  # Create a edit button with icon
+  #
+  # @param path [String] The path of the link.
```

Comment:
I finally got you.

But you were also expecting much more than I thought, initially these helpers were just wrappers and mostly to provide better code readability. I guess we will only wrote global helpers this way ?